RecoEcal/EgammaClusterProducers/src/PFECALSuperClusterProducer.cc

```diff
@@ -368,5 +389,60 @@ void PFECALSuperClusterProducer::fillDescriptions(edm::ConfigurationDescriptions
+    psd1.add<uint>("nClusterFeatures", 12);
+    psd1.add<uint>("nWindowFeatures", 18);
+    psd1.add<uint>("maxNClusters", 45);
+    psd1.add<uint>("maxNRechits", 40);
+    psd1.add<uint>("batchSize", 64);
+    psd1.add<std::string>("collectionStrategy", "Cascade");
+    desc.add<edm::ParameterSetDescription>("deepSuperClusterGraphConfig", psd1);
+  }
+  descriptions.add("particleFlowSuperClusterECALDeepSC", desc);
+
+  // Mustache configuration
+  edm::ParameterSetDescription descM;
+  descM.add<std::string>("PFSuperClusterCollectionEndcap", "particleFlowSuperClusterECALEndcap");
+  descM.add<bool>("doSatelliteClusterMerge", false);
+  descM.add<double>("thresh_PFClusterBarrel", 0.0);
+  descM.add<std::string>("PFBasicClusterCollectionBarrel", "particleFlowBasicClusterECALBarrel");
+  descM.add<bool>("useRegression", true);
+  descM.add<double>("satelliteMajorityFraction", 0.5);
+  descM.add<double>("thresh_PFClusterEndcap", 0.0);
+  descM.add<edm::InputTag>("ESAssociation", edm::InputTag("particleFlowClusterECAL"));
+  descM.add<std::string>("PFBasicClusterCollectionPreshower", "particleFlowBasicClusterECALPreshower");
+  descM.add<bool>("use_preshower", true);
+  descM.addUntracked<bool>("verbose", false);
+  descM.add<double>("thresh_SCEt", 4.0);
+  descM.add<double>("etawidth_SuperClusterEndcap", 0.04);
+  descM.add<double>("phiwidth_SuperClusterEndcap", 0.6);
+  descM.add<bool>("useDynamicDPhiWindow", true);
+  descM.add<std::string>("PFSuperClusterCollectionBarrel", "particleFlowSuperClusterECALBarrel");
+  descM.add<edm::ParameterSetDescription>("regressionConfig", SCEnergyCorrectorSemiParm::makePSetDescription());
+  descM.add<bool>("applyCrackCorrections", false);
+  descM.add<double>("satelliteClusterSeedThreshold", 50.0);
+  descM.add<double>("etawidth_SuperClusterBarrel", 0.04);
+  descM.add<std::string>("PFBasicClusterCollectionEndcap", "particleFlowBasicClusterECALEndcap");
+  descM.add<edm::InputTag>("PFClusters", edm::InputTag("particleFlowClusterECAL"));
+  descM.add<double>("thresh_PFClusterSeedBarrel", 1.0);
+  descM.add<std::string>("ClusteringType", "Mustache");
+  descM.add<std::string>("EnergyWeight", "Raw");
+  descM.add<edm::InputTag>("BeamSpot", edm::InputTag("offlineBeamSpot"));
+  descM.add<double>("thresh_PFClusterSeedEndcap", 1.0);
+  descM.add<double>("phiwidth_SuperClusterBarrel", 0.6);
+  descM.add<double>("thresh_PFClusterES", 0.0);
+  descM.add<bool>("seedThresholdIsET", true);
+  descM.add<bool>("isOOTCollection", false);
+  descM.add<edm::InputTag>("barrelRecHits", edm::InputTag("ecalRecHit", "EcalRecHitsEB"));
+  descM.add<edm::InputTag>("endcapRecHits", edm::InputTag("ecalRecHit", "EcalRecHitsEE"));
+  descM.add<std::string>("PFSuperClusterCollectionEndcapWithPreshower",
+                         "particleFlowSuperClusterECALEndcapWithPreshower");
+  descM.add<bool>("dropUnseedable", false);
+  descriptions.add("particleFlowSuperClusterECALMustache", descM);
```

Comment:
could you put this as the first one and the DeepSC algorithm as the second one ?
otherwise it's quite hard to spot the differences (if any) in the configuration of the Mustache algorithm.